predictable-app-updating.md

```diff
@@ -4,34 +4,41 @@ Author: Dibyajyoti Pal (dibyapal@chromium.org)
 
 # **Introduction**
 
-This explainer proposes a way to have PWAs fully update their identities in a safe and resourceful manner, to further bridge the gap between PWAs and native apps. This is done by updating the [manifest spec](https://www.w3.org/TR/appmanifest/#web-application-manifest) to have a specific field for making updates more deterministic by the developer, and for providing a consistent experience. The proposal attempts to do so in a way that:
+This explainer proposes a way to have PWAs fully update their identities in a safe and resourceful manner, to further bridge the gap between PWAs and native apps. This is done by updating the [manifest spec](https://www.w3.org/TR/appmanifest/#web-application-manifest) to have a specific field for making updates more deterministic by the developer, and for providing a consistent experience across different user agents. The proposal attempts to do so in a way that:
 
 1. Uses less resources, making network usage more efficient.  
 2. Prevent user confusion by showing the update UX less often.
+3. Allows user agents who haven't yet implemented manifest updating to do so in a different way.
+
+One thing to note here is that installed PWAs are inherently different from how the rest of the Web Platform works. While the origin is the security boundary of the Web Platform, the installed experience has [different security parameters](https://www.w3.org/TR/appmanifest/#dfn-security-sensitive-members).
```

Comment:
Why are you saying this? What was the root of that message they sent? They were saying that it's weird to need a token for something like this on the web, why not just make it so icon updates don't happen unless the URL changes?

The message here is that manifest properties that are security sensitive<put link> are shown by the user agent instead of an origin, so changes to these properties must be handled carefully and the manifest update algorithm<link> already specifies that UX must be shown to the user when they do.

The point of this explainer (please make sure it's presented as such) is that this is specifying 'when' to consider update, and specifying it in a way that accomplishes the above 2 goals.